homeassistant/components/ambient_network/sensor.py

```diff
@@ -0,0 +1,310 @@
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key=TYPE_LASTRAIN,
+        translation_key="last_rain",
+        icon="mdi:water",
+        device_class=SensorDeviceClass.TIMESTAMP,
+    ),
+    SensorEntityDescription(
+        key=TYPE_LIGHTNING_PER_DAY,
+        translation_key="lightning_strikes_per_day",
+        icon="mdi:lightning-bolt",
+        native_unit_of_measurement="strikes",
+        state_class=SensorStateClass.TOTAL,
+        suggested_display_precision=0,
+    ),
+    SensorEntityDescription(
+        key=TYPE_LIGHTNING_PER_HOUR,
+        translation_key="lightning_strikes_per_hour",
+        icon="mdi:lightning-bolt",
+        native_unit_of_measurement="strikes",
+        state_class=SensorStateClass.TOTAL,
+        suggested_display_precision=0,
+    ),
+    SensorEntityDescription(
+        key=TYPE_LIGHTNING_DISTANCE,
+        translation_key="lightning_distance",
+        icon="mdi:lightning-bolt",
+        native_unit_of_measurement=UnitOfLength.MILES,
+        device_class=SensorDeviceClass.DISTANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_MAXDAILYGUST,
+        translation_key="max_daily_gust",
+        native_unit_of_measurement=UnitOfSpeed.MILES_PER_HOUR,
+        device_class=SensorDeviceClass.WIND_SPEED,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key=TYPE_MONTHLYRAININ,
+        translation_key="monthly_rain",
+        native_unit_of_measurement=UnitOfPrecipitationDepth.INCHES,
+        device_class=SensorDeviceClass.PRECIPITATION,
+        state_class=SensorStateClass.TOTAL,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_PM25_24H,
+        translation_key="pm25_24h_average",
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        device_class=SensorDeviceClass.PM25,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key=TYPE_PM25,
+        native_unit_of_measurement=CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        device_class=SensorDeviceClass.PM25,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key=TYPE_SOLARRADIATION,
+        translation_key="solar_radiation",
+        native_unit_of_measurement=UnitOfIrradiance.WATTS_PER_SQUARE_METER,
+        device_class=SensorDeviceClass.IRRADIANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key=TYPE_TEMPF,
+        translation_key="temperature",
+        native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
+        device_class=SensorDeviceClass.TEMPERATURE,
```

Comment:
These 2 can have their translation key removed to make use of device class translations. This will also keep naming consistent with other inetgrations

Reply:
Removed.